extensions/wikia/Search/classes/Test/ResultTest.php

```diff
@@ -394,34 +396,30 @@ public function testReplaceUnusualEscapes() {
 			'100%25+Completion',
 			Result::replaceUnusualEscapes( urlencode( '100% Completion' ) )
 		);
-
 	}
 
 	/**
 	 * @group Slow
 	 * @slowExecutionTime 0.10111 ms
-	 * @covers Wikia\Search\Result::getVideoViews
+	 * @covers \Wikia\Search\Result::getVideoViews
 	 */
 	public function testGetVideoViews() {
-		$mockResult = $this->getMockBuilder( 'Wikia\Search\Result' )
-			->setConstructorArgs( array( array( 'pageid' => 123 ) ) )
-			->setMethods( null )
-			->getMock();
-		$mockService = $this->getMockBuilder( 'Wikia\Search\MediaWikiService' )
-			->disableOriginalConstructor()
-			->setMethods( array( 'getFormattedVideoViewsForPageId' ) )
-			->getMock();
+		$mockResult =
+			$this->getMockBuilder( 'Wikia\Search\Result' )->setConstructorArgs( [ [ 'pageid' => 123 ] ] )->setMethods(
+					null
+				)->getMock();
+		$mockService =
+			$this->getMockBuilder( 'Wikia\Search\MediaWikiService' )->disableOriginalConstructor()->setMethods(
+					[ 'getFormattedVideoViewsForPageId' ]
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getFormattedVideoViewsForPageId" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)